web/client/components/search/SearchBarUtils.js

```diff
@@ -6,6 +6,22 @@
  * LICENSE file in the root directory of this source tree.
 */
 
+const getTotalMaxResults = (searchOptions, maxResults) => {
+    let array1 = [];
```

Comment:
try not to use generic names, but others a bit more specific like 
allMaxFeatures